src/components/Summary/AlterationSummary.vue

```diff
@@ -111,7 +111,7 @@
               :currentJsDate="getCurrentJsDate"
               :effectiveDateTime="getEffectiveDateTime"
               @dateTimeString="setEffectiveDateTimeString($event)"
-              @isFutureEffective="setIsFutureEffective($event)"
+              @isFutureEffective="setIsFutureEffective($event); emitHaveChanges()"
```

Comment:
I could have put this emitted on the other 2 events here as well, since they all work together to set the immediate/future-effective state, but by having it on just this event, the extra fee toggles on/off when the user changes the form value that affects the fee, which is nice and reactive (instant feedback feels right).